File: AkkaWithZeroMQ/src/main/java/org/akka/essentials/zeromq/example3/client/MyClientActorSystem.java
package org.akka.essentials.zeromq.example3.client;

import akka.actor.ActorSystem;
import akka.actor.Props;
import akka.routing.RoundRobinRouter;

public class MyClientActorSystem {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		ActorSystem system = ActorSystem.create("zeromqClientTest");
		system.actorOf(new Props(ClientActor.class)
				.withRouter(new RoundRobinRouter(3)), "client");
	}

}

File: AkkaWithZeroMQ/src/main/scala/org/akka/essentials/zeromq/example3/client/MyClientActorSystem.scala
package org.akka.essentials.zeromq.example3.client
import akka.actor.ActorSystem
import akka.actor.Props
import akka.routing.RoundRobinRouter

object MyClientActorSystem {

  def main(args: Array[String]): Unit = {
    val system = ActorSystem("zeromqTest")
    system.actorOf(Props[ClientActor].withRouter(
      RoundRobinRouter(nrOfInstances = 3)), name = "client")
  }
}